data/src/main/java/com/hous/data/source/remote/RemoteRulesTodayDataSourceImpl.kt

```diff
@@ -17,41 +17,96 @@ class RemoteRulesTodayDataSourceImpl @Inject constructor(
+                    response.data?.homeRuleCategories!!.map { it.toCategoryInfo() },
+                    response.data.todayTodoRules.map { it.toRuleInfo() }
+                )
+            },
+            {
+                Timber.e(it.message)
+                return null
+            }
+        )
 
     override suspend fun getTempManagerInfoList(
         roomId: String,
         ruleId: String
-    ): WrapperClass<TempManagerResponse> {
-        return rulesApi.getTempManagerInfoList(ROOM_ID, ruleId)
-    }
+    ): TempManagerInfo? =
+        runCatching {
+            rulesApi.getTempManagerInfoList(ROOM_ID, ruleId)
+        }.fold(
+            { response ->
+                return TempManagerInfo(
+                    response.data!!.id,
+                    response.data.homies.map { it.toHomieInfo() }
+                )
+            },
+            {
+                Timber.e(it.message)
+                return null
+            }
+        )
 
     override suspend fun putTempManagerInfoList(
         roomId: String,
         ruleId: String,
         tmpRuleMembers: TempManagerRequest
-    ): WrapperClass<Any> {
-        return rulesApi.putTempManagerInfoList(ROOM_ID, ruleId, tmpRuleMembers)
+    ): Any? {
+        return runCatching {
+            rulesApi.putTempManagerInfoList(
+                ROOM_ID,
+                ruleId,
+                tmpRuleMembers
+            )
+        }.fold(
+            {}, {
+            Timber.e(it.message)
+            null
+        }
+        )
     }
 
-    override suspend fun getMyToDoInfoList(roomId: String): WrapperClass<List<Rule>> {
-        return rulesApi.getMyTodoInfoList(ROOM_ID)
+    override suspend fun getMyToDoInfoList(roomId: String): List<RuleInfo>? {
+        return runCatching { rulesApi.getMyTodoInfoList(ROOM_ID) }.fold(
+            { response ->
+                response.data?.map { it.toRuleInfo() }
+            },
+            {
+                Timber.e(it.message)
+                null
+            }
+        )
     }
 
     override suspend fun putMyToDoCheckLust(
         roomId: String,
         ruleId: String,
         isCheck: MyToDoCheckRequest
-    ): WrapperClass<MyToDoCheckRequest> {
-        return rulesApi.putMyToDoCheckList(ROOM_ID, ruleId, isCheck)
+    ) {
+        runCatching { rulesApi.putMyToDoCheckList(ROOM_ID, ruleId, isCheck) }.onFailure {
+            Timber.e(it.message)
+        }
     }
 
     override suspend fun getRuleTableInfoList(
         roomId: String,
         categoryId: String
-    ): WrapperClass<RulesTableResponse> {
-        return rulesApi.getRuleTableInfoList(ROOM_ID, categoryId)
-    }
+    ): RulesTableInfo? =
+        runCatching { rulesApi.getRuleTableInfoList(ROOM_ID, categoryId) }.fold(
+            { response ->
+                return RulesTableInfo(
+                    response.data?.keyRules!!.map { it.toRuleInfo() },
+                    response.data.rules.map { it.toRuleInfo() }
+                )
```

Comment:
```suggestion
                return RulesTableInfo(
                    response.data?.keyRules!!.map { keyRulesList -> keyRulesList.toRuleInfo() },
                    response.data.rules.map { rulesList -> rulesList.toRuleInfo() }
                )
```
우리 가독성을 위해 it 사용을 지양해 봅시다..!

Reply:
네 알겠습니다!! 가독성 진짜 중요하죵 ㅎ ㅎ ㅎ